partiql-spi/src/main/kotlin/org/partiql/spi/catalog/Session.kt

```diff
@@ -43,7 +43,8 @@ public interface Session {
     public companion object {
 
         /**
-         * Returns a [Session] with only the "empty" catalog implementation.
+         * Returns a [Session] with only the "empty" catalog implementation. Note that this does NOT add the
+         * PartiQL System Catalog.
          */
         @JvmStatic
         public fun empty(): Session = object : Session {
```

Comment:
Session.empty() is used in several places so would need the psystem available. I don't believe we need bare()  factory method either. I was referring to a builder method in the previous PR, not a factory.

Think about it this, you have a named method for which builder either adds the system or not.

```
Session.builder()....build()   // standard builder comes with standard system
Session.system(catalog)...builder()  // .system() is a named *builder* factory which replaces the standard system.
```

Reply:
Yeah, the latest revision has this. I can just revert the last commit (the addition of the bare method).

Reply:
I just reverted and added the PartiQL system catalog to `Session.empty()`. See the last three commits.